backend/flask_api.py

```diff
@@ -209,9 +209,9 @@ def update_project(project_id):
         res = coll.find_one({'_id': project_id})
         if res is None:
             raise ApiException("Project not found", 404)
-        elif request.is_json or request.headers['content-type'] == "application/json5":
+        elif request.is_json or request.content_type == "application/json5;charset: utf-8":
```

Comment:
you have to check for:
`elif request.is_json or 'application/json5' in request.content_type:`

specifying the charset or other things in the header content is optional, but if you check for == (equals) you will not catch any variance there